acme/acme/client.py

```diff
@@ -255,16 +258,23 @@ def retry_after(cls, response, default):
         :rtype: `datetime.datetime`
 
         """
-        retry_after = response.headers.get('Retry-After', str(default))
+        retry_after = response.headers.get('Retry-After', str(default)).strip()
```

Comment:
Done; I'm not sure if requests passes along trailing whitespace but the HTTP spec does seem to allow it.